src/knex-store.ts

```diff
@@ -166,6 +166,17 @@ function knex_store(this: any, options: Options) {
     }
   )
 
+  seneca.add(
+    'sys:entity,transaction:adopt',
+    async function (msg: any, reply: any) {
+      const trxProvider = db.transactionProvider()
```

Comment:
this assumes we use the same db connection as the outside world, which may not be the case
instead, pass the active transaction into this message as parameter:
see: https://github.com/senecajs/seneca-entity/blob/master/src/entity.ts#L286

Reply:
I don't know if I understand, I must pass only the Knex instance to the get-handle, right?
Because the `db` variable is the Knex instance without the database connection settings.
I did the same thing as was done to start transactions using seneca.entity.begin() but now I'm not sure if it was right, see:
seneca
```
seneca.add('sys:entity,transaction:begin', function transactionBegin(msg, reply) {
			knex.transaction().then((trx) => {
				reply({ get_handle: ()=>(trx) });
			});
		})
```